public/keymaps/m/mlego_m48_rev1_default.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard": "mlego/m48/rev1",
+    "keymap": "default",
+    "commit": "d02be63000a4f5efe728f36608c33bcdaee08330",
+    "layers": [
+        ["KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_BSPC", "KC_ESC", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_ENT", "KC_TRNS", "KC_LCTL", "KC_LALT", "KC_LGUI", "TT(_LWR)", "KC_SPC", "KC_SPC", "TT(_RSE)", "KC_LEFT", "KC_DOWN", "KC_UP", "KC_RGHT"],
```

QMK Configurator doesn't have access to any defined layer names – all layers are numbered only, so you have to reference the layers by their numbers in layer switching keycodes.

```suggestion
        ["KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_BSPC", "KC_ESC", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_ENT", "KC_TRNS", "KC_LCTL", "KC_LALT", "KC_LGUI", "TT(1)", "KC_SPC", "KC_SPC", "TT(2)", "KC_LEFT", "KC_DOWN", "KC_UP", "KC_RGHT"],
```